manifests/0000_51_olm_02_operator_clusterrole.yaml

```diff
@@ -167,3 +167,8 @@ rules:
     - operator-controller-metrics-reader
     - operator-controller-proxy-role
     - operator-controller-leader-election-role
+    - bundle-reader
+    - bundle-uploader
+    - core-admin
+    - helm-provisioner-admin
```

Comment:
Fine for now, but in upstream, we should make sure these (and all rukpak install objects) are all prefixed with `rukpak-` IMO.